armada/handlers/armada.py

```diff
@@ -289,9 +292,18 @@ def sync(self):
                                                 wait=chart_wait,
                                                 timeout=chart_timeout)
 
+
                 LOG.debug("Cleaning up chart source in %s",
                           chartbuilder.source_directory)
 
+                if test_chart:
```

Comment:
Does helm store the testing state anywhere internally? Can this also be returned when asking armada for a list of releases (e.g. successfully installed, and successfully ran tests)?